webpack.config.js

```diff
@@ -55,7 +55,7 @@ module.exports = (env, argv) => {
         module: {
             rules: [
                 {
-                    test: /\.js$/,
+                    test: /\.jsx?$/,
```

Comment:
`.jsx` files weren't being picked up in the Webpack configs, this will add both `.js` and `.jsx` files.